src/coreclr/gc/gcpriv.h

```diff
@@ -419,8 +419,10 @@ enum gc_oh_num
     soh = 0,
     loh = 1,
     poh = 2,
-    none = 3,
-    total_oh_count = 4
+    free = 3,
+    none = 4,
+    total_oh_count = 3,
+    total_bucket_count = 5
```

Comment:
> total_bucket_count = 5 [](http://example.com/codeflow?start=3&length=23)

I would actually take this out of the gc_oh_num and make it a global variable like recorded_committed_bucket_count